source/frontend/src/features/documents/list/DocumentListContainer.tsx

```diff
@@ -47,8 +48,7 @@ const DocumentListContainer: React.FunctionComponent<
     if (documents !== undefined && isMounted()) {
       setDocumentResults([
         ...documents
-          .map(x => x.document)
-          .filter((x): x is Api_Document => !!x)
+          .filter((x): x is Api_DocumentRelationship => !!x?.document)
```

Comment:
@FuriousLlama this seems odd to me. The onDelete and other methods seem to be assuming that an Api_DocumentRelationship is what each row is based on, but it used to be that each DocumentRow was equivalent to an Api_Document, and there is no way of going from a DocumentRow to an Api_DocumentRelationship

Reply:
Looks like the types for these are not being validated.
![image](https://user-images.githubusercontent.com/12201292/216153653-aa40effc-1d4c-4fc0-8785-b9d648c90be1.png)

My guess is that the original list only cared about documents and not document relationships which makes sense. So if we want to keep that distinction the onDelete callback would need to find the documentrelationship based on the document id. Otherwise need to convert the list to work with relationshipdocs

Reply:
hmm, ok. I think right now we never delete a document without a relationship but maybe we will need this in the future. Sounds like a tech debt item.